forte/data/base_pack.py

```diff
@@ -791,13 +791,13 @@ def get_ids_by_creator(self, component: str) -> Set[int]:
         return entry_set
 
     def is_created_by(
-        self, entry: Entry, components: Union[str, Iterable[str]]
+        self, entry_tid: int, components: Union[str, Iterable[str]]
```

Comment:
For public interfaces, I think it's better to take care of backward compatibility.  For example, maybe we can make it `Union[Entry, int]`.